src/frontends/tensorflow_common/include/place.hpp

```diff
@@ -33,6 +33,9 @@ class Place : public ov::frontend::Place {
     std::vector<std::string> get_names() const override {
         return m_names;
     }
+    void set_names(const std::vector<std::string>& names) {
```

Comment:
do we need `override`?

Reply:
no

Reply:
This method is a sort of WA for TF Lite. I think we need to leave a comment here for us to avoid from such method in the future.